src/events/messageReactionRemove.ts

```diff
@@ -0,0 +1,11 @@
+import { onEvent } from '../helpers/onEvent.js';
+import { allReactionHandlers } from '../reactionHandlers/remove.js';
+import { buildExecute } from './messageReaction.js';
+
+/**
+ * The event handler for emoji reactions.
+ */
+export const messageReactionRemove = onEvent('messageReactionRemove', {
+	once: false,
+	execute: buildExecute(allReactionHandlers),
+});
```

Comment:
Feels weird for the `'messageReactionRemove'` event to execute `allReactionHandlers`, instead of just a `remove` handler..